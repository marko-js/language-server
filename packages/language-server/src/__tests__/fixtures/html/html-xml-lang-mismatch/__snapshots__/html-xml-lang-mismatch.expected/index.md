## Diagnostics
### Ln 1, Col 17
```marko
> 1 | <html lang="en" xml:lang="es">
    |                 ^^^ Argument of type '{ lang: string; xml: string; renderBody: () => MarkoReturn<void>; }' is not assignable to parameter of type 'Directives & Marko·Inputᐸʺhtmlʺᐳ'.
  Object literal may only specify known properties, and '"xml"' does not exist in type 'Directives & Marko·Inputᐸʺhtmlʺᐳ'.
  2 |   <head>
  3 |     <title>Conflicting Languages</title>
  4 |   </head>
```

### Ln 1, Col 2
```marko
> 1 | <html lang="en" xml:lang="es">
    |  ^^^^ Fix all of the following:
  Lang and xml:lang attributes do not have the same base language
  2 |   <head>
  3 |     <title>Conflicting Languages</title>
  4 |   </head>
```

