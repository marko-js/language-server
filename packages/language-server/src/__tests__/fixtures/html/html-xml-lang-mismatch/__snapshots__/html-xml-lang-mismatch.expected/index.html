<html data-marko-node-id="0" lang="en" xml:lang="es">
  <head data-marko-node-id="1">
    <title data-marko-node-id="2">Conflicting Languages</title>
  </head>
</html>